modules/Milestone/MilestoneController.js

```diff
@@ -6,13 +6,31 @@ const calendlyService = require('../../services/calendlyService');
 const nearService = require('../../services/nearService');
 const { reportError } = require('../../services/errorReportingService');
 const logger = require('../../utilities/logger');
+const grantConfig = require('../../config/grant');
 
 /**
  * MilestoneController.js
  *
  * @description :: Server-side logic for managing Milestones.
  */
 module.exports = {
+  async create(req, res) {
+    try {
+      if (!grantConfig.allowMilestonesOnTheGo) {
+        res.status(400).json({
+          message: 'New milestones cannot be created',
+        });
+        return;
+      }
```

Comment:
What is functionality when grantConfig.allowMilestonesOnTheGo is true? Is empty return the desired behavior

Reply:
Basically the default is `allowMilestonesOnTheGo = false`
When it's false the milestones part of the grant request are fixed and defined in the first form filled;
When it's true you can continuously add milestones one after the other, it was a request from NEARWEEK.